Math.enum.h

```diff
@@ -19,20 +19,22 @@
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-/**
- * @file
- * Includes Math's enums.
- */
+// @file
+// Includes Math's enums.
 
 // Prevents processing this includes file for the second time.
 #ifndef MATH_ENUM_H
 #define MATH_ENUM_H
 
 #ifndef __MQLBUILD__
 //
-// Data type identifiers.
-// @docs
-// - https://www.mql5.com/en/docs/constants/indicatorconstants/enum_datatype
+/** Data type identifiers.
+  * @docs
+  * - https://www.mql5.com/en/docs/constants/indicatorconstants/enum_datatype
+  * /
+/**
+  * Enum datatype
```

Comment:
This block is not necessary, as the previous block is enough. We don't need two.